sound/soc/intel/common/soc-acpi-intel-cnl-match.c

```diff
@@ -37,6 +42,14 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_cnl_machines[] = {
 		.sof_fw_filename = "sof-cnl.ri",
 		.sof_tplg_filename = "sof-cml-rt5682.tplg",
 	},
+	{
+		.id = "MX98357A",
+		.drv_name = "sof_rt5682",
```

Comment:
maybe good to use an alias so that we can always use a different machine driver if needed?